ambry-api/src/main/java/com/github/ambry/config/FrontendConfig.java

```diff
@@ -219,6 +228,20 @@
   @Default(DEFAULT_ACCOUNT_STATS_STORE_FACTORY)
   public final String accountStatsStoreFactory;
 
+  /**
```

Comment:
What's the reason for making these two configurable?

Reply:
In close source we have this cfg.containerMetricsEnabledRequestTypes which determine which metric need to emit container level metric. And in Container Metric we are hardcode the request type which enable the container metric. So if we update the open source, it's easy to forget modify the close source. So I think it's better if in close source we can get the containerMetricsEnabledRequestTypes from frontend config as well. In fact, we do forget to add UndeleteBlob and PutBlob in cfg.containerMetricsEnabledRequestTypes. And for the second config is to determine if the request is get or put type, in order to aggregate the per request metric to one put and one get metric since Nuage can only support reading two simple metric for live traffic check.